MorphicCore.Tests/KeychainTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.ComTypes;
+using System.Text;
+using Xunit;
+
+namespace MorphicCore.Tests
+{
+    public class KeychainTests : IDisposable
+    {
+        public KeychainTests()
+        {
+            directoryName = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
+        }
+
+        private string directoryName;
+        
+        [Fact]
+        public void TestSaveLoad()
+        {
+            var encrypt = new MockEncrypter();
+            var ko = new KeychainOptions();
```

Comment:
This one may seem nitpicky, but will help keep the codebase consistent.  Here and elsewhere, can you use unabbreviated names like `options` or `keychainOptions`.  A few lines down use `keychain` instead of `kc`, etc.